crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -392,23 +444,23 @@ def openstackbackup
             "Failed to create OpenStack database dump: #{db_dump[:stdout_and_stderr]}"
           )
           FileUtils.rm_f(dump_path)
-          raise ::Crowbar::Error::UpgradeDatabaseDumpError.new(
+          raise ::Crowbar::Error::Upgrade::DatabaseDumpError.new(
             db_dump[:stdout_and_stderr]
           )
         end
 
         ::Crowbar::UpgradeStatus.new.end_step
-      rescue ::Crowbar::Error::UpgradeNotEnoughDiskSpaceError => e
+      rescue ::Crowbar::Error::Upgrade::NotEnoughDiskSpaceError => e
         ::Crowbar::UpgradeStatus.new.end_step(
           false,
           backup_openstack: {
             data: e.message,
             help: "Make sure you have enough disk space to store the OpenStack database dump."
           }
         )
-      rescue ::Crowbar::Error::UpgradeFreeDiskSpaceError,
-             ::Crowbar::Error::UpgradeDatabaseSizeError,
-             ::Crowbar::Error::UpgradeDatabaseDumpError => e
+      rescue ::Crowbar::Error::Upgrade::FreeDiskSpaceError,
+             ::Crowbar::Error::Upgrade::DatabaseSizeError,
+             ::Crowbar::Error::Upgrade::DatabaseDumpError => e
         ::Crowbar::UpgradeStatus.new.end_step(
```

Comment:
it's because i started the PR when the openstackbackup PR was still not merged. fixed